tests/integration/test_script_runner.py

```diff
@@ -384,8 +383,7 @@ def _execute_integration_test(  # noqa: C901, PLR0915
         base_dir = pathlib.Path(file_relative_path(__file__, "../../"))
         os.chdir(base_dir)
         # Ensure GX is installed in our environment
-        installed_packages = [pkg.key for pkg in pkg_resources.working_set]
-        if "great-expectations" not in installed_packages:
+        if importlib.util.find_spec("great-expectations") is None:
```

Comment:
I think we should probably just die here instead. Requiring installation of GX before running integration tests seems reasonable.